package.json

```diff
@@ -63,7 +63,9 @@
   ],
   "scripts": {
     "build": "rollup -c",
-    "test": "npm run build && grunt test",
+    "test-node": "jest --verbose --environment=node",
```

Comment:
Hm, not sure here. Personally I have a favor to use `test:node` and `test:web`.

Reply:
Changed.

Reply:
Updated